src/app/games/trexo/TrexoTutorial.ts

```diff
@@ -0,0 +1,95 @@
+import { Tutorial, TutorialStep } from 'src/app/components/wrapper-components/tutorial-game-wrapper/TutorialStep';
+import { Coord } from 'src/app/jscaip/Coord';
+import { MGPValidation } from 'src/app/utils/MGPValidation';
+import { TrexoMove } from './TrexoMove';
+import { TrexoRules } from './TrexoRules';
+import { TrexoPiece, TrexoPieceStack, TrexoState } from './TrexoState';
+import { Player } from 'src/app/jscaip/Player';
+
+const ______: TrexoPieceStack = TrexoPieceStack.EMPTY;
+const ONE__0: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ONE, 1, 0)]);
+const ZERO_0: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ZERO, 1, 0)]);
+const ONE__1: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ONE, 1, 1)]);
+const ZERO_1: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ZERO, 1, 1)]);
+const ZERO_2: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ZERO, 1, 2)]);
+const ONE__2: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ONE, 1, 2)]);
+const ZERO_3: TrexoPieceStack = TrexoPieceStack.from([
+    new TrexoPiece(Player.ZERO, 1, 0),
+    new TrexoPiece(Player.ZERO, 2, 3),
+]);
+const ONE__3: TrexoPieceStack = TrexoPieceStack.from([
+    new TrexoPiece(Player.ONE, 1, 1),
+    new TrexoPiece(Player.ONE, 2, 3),
+]);
+const ZERO_4: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ZERO, 1, 4)]);
+const ONE__4: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ONE, 1, 4)]);
+const ZERO_5: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ZERO, 1, 5)]);
+const ONE__5: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ONE, 1, 5)]);
+const ONE__6: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ONE, 1, 6)]);
+const ZERO_6: TrexoPieceStack = TrexoPieceStack.from([new TrexoPiece(Player.ZERO, 1, 6)]);
+
+export class TrexoTutorial extends Tutorial {
+    public tutorial: TutorialStep[] = [
+        // 1. but du jeu et plateau
+        TutorialStep.informational(
+            $localize`Goal of the game`,
+            $localize`At Trexo, the goal of the game is to align 5 pieces of your color in a vertical, horizontal or diagonal line. However, the players play with tiles that are constituted of two pieces, one from each player!`,
+            TrexoState.getInitialState(),
+        ),
+        TutorialStep.anyMove(
+            $localize`Dropping a tile`,
+            $localize`When you drop a tile, it needs to be on even ground, and it cannot be right on top of another tile. In others words, it needs to be either on the floor, or on two tiles at the same height. To do that, just click on the square where you want to put the opponent side of the tile, then on the neighboring square where you want to put your piece.<br/><br/>You're playing Dark, put a tile on the board.`,
```

Comment:
To do that -> to drop a tile (il y a une phrase entre le "that" et ce à quoi il réfère, c'est trop)